lib/circles-common/src/http/auth.rs

```diff
@@ -127,7 +124,7 @@ impl Service for AuthenticatorService {
                         box db_future
                     }
                     Err(e) => {
-                        debug!("attempted unathorized access to {}", uri);
+                        debug!("attempted unathorized access to {}", req.path());
```

Comment:
When did the req.path() happen and what is the difference with uri()?

Reply:
req.path() is the part of Uri that goes after domain

In http://localhost:8080/endroint, `/endpoint` is the path.